src/lobster/model/modern_bert/_modern_bert.py

```diff
@@ -136,6 +136,30 @@ def configure_optimizers(self):
 
         return {"optimizer": optimizer, "lr_scheduler": scheduler}
 
+    def tokens_to_latents(self, input_ids: torch.Tensor, attention_mask: torch.Tensor) -> torch.Tensor:        
```

Comment:
this will be a commonly used method - can you add a short docstring with an amino acid and smiles example?